packages/docusaurus-theme-classic/src/theme/NavbarItem/styles.module.css

```diff
@@ -6,7 +6,7 @@
  */
 
 .iconExternalLink {
-  margin-left: 0.3rem;
+  margin-left: 0.3em;
```

Comment:
Why?

Reply:
copy past from different icons, i'm going to revert this